.github/workflows/publish_charm.yaml

```
@@ -48,25 +110,35 @@ jobs:
           rm -rf $TEMP_DIR
       - name: Install charmcraft
         run: sudo snap install charmcraft --classic
-      - name: Log in to the Container registry
-        uses: docker/login-action@v2
+      - name: Download docker artifacts
+        uses: dawidd6/action-download-artifact@v2
+        with:
+          github_token: ${{secrets.GITHUB_TOKEN}}
```

```suggestion
          github_token: ${{ secrets.GITHUB_TOKEN }}
```
just added spacing :D

---

Added. Thx